pom.xml

```diff
@@ -112,18 +112,19 @@
         
         <hikari-cp.version>3.4.2</hikari-cp.version>
         <commons-dbcp2.version>2.2.0</commons-dbcp2.version>
-        
-        <junit.version>4.13.2</junit.version>
+
+        <junit5.version>5.9.1</junit5.version>
+        <junit4.version>4.13.2</junit4.version>
         <hamcrest.version>1.3</hamcrest.version>
-        <mockito.version>4.5.1</mockito.version>
+        <mockito.version>4.8.0</mockito.version>
         
         <!-- Plugin versions -->
         <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <maven-javadoc-plugin.version>3.3.0</maven-javadoc-plugin.version>
         <maven-resources-plugin.version>3.3.0</maven-resources-plugin.version>
         <maven-jar-plugin.version>3.2.0</maven-jar-plugin.version>
-        <maven-surefire-plugin.version>2.18.1</maven-surefire-plugin.version>
+        <maven-surefire-plugin.version>3.0.0-M7</maven-surefire-plugin.version>
```

Comment:
Is the M7 version stable?

Reply:
- I believe `3.0.0-M7` is stable, but I don't know why the `maven-surefire-plugin` community is still using milestone versions.  

- `Junit Platform` requires unit tests to be performed in `GraalVM Native Image`, `maven-surefire-plugin` version should be `>= 2.22.0`, but if we still use the version `< 3.0.0-M5`, we need to manually manage the version number of `JUnit Platform`, I  I think this operation of managing dependencies is unnecessary, refer to https://graalvm.github.io/native-build-tools/latest/maven-plugin.html#testing-support-version-compatibility .